README.md

```diff
@@ -104,6 +104,12 @@ First, install ocagent if you haven't.
 $ go get github.com/census-instrumentation/opencensus-service/cmd/ocagent
 ```
 
+Alternatively you can build ocagent from binary (from the root of your repo):
+
+```shell
+$ export GO111MODULE=on && make agent
```

Comment:
Setting `GO111MODULE=on` can trip out some builds so perhaps
```shell
$ GO111MODULE=on make agent
```
which is the same remedy I had recommended for him offline.

Reply:
I see, I moved setting `GO111MODULE` to Makefile instead, which should work the same way.